src/cmd/index.js

```diff
@@ -3,5 +3,6 @@ import build from './build';
 import lint from './lint';
 import run from './run';
 import sign from './sign';
+import newCommand from './new';
 
-export default {build, lint, run, sign};
+export default {build, lint, run, sign, newCommand};
```

Comment:
I agree `web-ext init` is a better option, I'll proceed with this changes. Unless @kumar303 disagrees.